### PR TITLE
New update

### PR DESCRIPTION
**Pull request by Janamou**
cc @filiph @kwalrath 

Thx for review!

### CODE REVIEW

File: 4-loaddatafromfirebase/firebase_counter/lib/src/todo_list/todo_list_service.dart [hunk 1]
```diff
@@ -1,6 +1,9 @@
+// Copyright (c) 2017, janamou. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
 import 'dart:async';
 
-import 'package:angular2/core.dart';
+import 'package:angular/core.dart';
```
Comment (kwalrath) [inline 4-loaddatafromfirebase/firebase_counter/lib/src/todo_list/todo_list_service.dart]: This should import `angular/angular.dart` (GLOBAL).

All angular/* imports should change to `angular/angular.dart`. (No core, no common, no browser, ...)

File: 6-addmaterialcomponents/firebase_counter/lib/app_component.html [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <material-fab (trigger)="dislike()">
-    <glyph icon="thumb_down"></glyph>
+    <material-icon icon="thumb_down"></material-icon>
```
Comment (kwalrath) [inline 6-addmaterialcomponents/firebase_counter/lib/app_component.html:2]: Does this work without `materialIcon` in the `directives` list?

**Review by kwalrath — COMMENTED** [review]
A couple of little issues, but otherwise LGTM. Thanks, Jana!

File: 4-loaddatafromfirebase/firebase_counter/lib/src/todo_list/todo_list_service.dart (hunk 1 above)
Comment (Janamou) [reply 4-loaddatafromfirebase/firebase_counter/lib/src/todo_list/todo_list_service.dart]: @kwalrath ok, I will fix this but I created a new project in webstorm so the template should be updated as well ;-)

File: 6-addmaterialcomponents/firebase_counter/lib/app_component.html (hunk 2 above)
Comment (Janamou) [reply 6-addmaterialcomponents/firebase_counter/lib/app_component.html:2]: @kwalrath it worked :-D but ok, lets rather put it there (I am going to do that) :-D

**Comment by Janamou** [comment]
@kwalrath done :-)

File: 6-addmaterialcomponents/firebase_counter/lib/app_component.html (hunk 2 above)
Comment (kwalrath) [reply 6-addmaterialcomponents/firebase_counter/lib/app_component.html:2]: That's weird that it worked! It wasn't working for me, and I didn't see a change in angular_components. I'm going to look into this a bit.

Comment (kwalrath) [reply 6-addmaterialcomponents/firebase_counter/lib/app_component.html:2]: My bad, it looks like MaterialIconComponent made it into [materialDirectives](https://webdev.dartlang.org/api/angular_components/angular_components/materialDirectives-constant) in time for 0.6.0.

**Review by kwalrath — APPROVED** [review]
I'm going to commit this, since it works, but we should probably remove the MaterialIconComponent reference from directives.

**Pull request merged**